_includes/join-hero.html

```diff
@@ -2,13 +2,18 @@
   <div class="secondary-hero-container">
     <div class="secondary-hero-contents">
       <div class="secondary-hero-text">
-        <h4 class="secondary-hero-subheading--desktop">How to join</h4>
-        <h4 class="secondary-hero-subheading--mobile">How to join</h4>
-        <h1 class="secondary-hero-heading">How we start projects and get things done</h1>
+        <h4 class="secondary-hero-subheading--desktop">Join Us</h4>
+        <h4 class="secondary-hero-subheading--mobile">Join Us</h4>
+        <h1 class="secondary-hero-heading">Be part of the change in your commmunity</h1>
+        <a class="join-button"
+          href="https://docs.google.com/presentation/d/1_7zd36CeHqi15DTqL3Wmh3dLCiN8EVbfW2iy1j-gs0M/edit?usp=sharing"
+          target="_blank">Start
```

Comment:
Let's add `rel="noopener"` below this. For more information check this article out: https://techstacker.com/html-noopener-attribute/